test/oauth-server/service/token.go

```diff
@@ -45,7 +45,7 @@ func makeAccessToken(clientID, host, deviceID string, issuedAt, expires time.Tim
 			return nil, fmt.Errorf("failed to set %v: %w", jwt.ExpirationKey, err)
 		}
 	}
-	if err := token.Set(TokenScopeKey, []string{"openid", "r:deviceinformation:*", "r:resources:*", "w:resources:*", "w:subscriptions:*"}); err != nil {
+	if err := token.Set(TokenScopeKey, scopes); err != nil {
```

Comment:
replace `TokenScopeKey` with https://github.com/plgd-dev/hub/blob/main/pkg/security/jwt/claims.go#L16

Reply:
maybe you need to convert it as an array if it contains contained scope with "," separator

Reply:
scopes is of type string, already in authorize.go defined as space separated scope list.

Reply:
ok so you need to split to array by " "

Reply:
No, because then you return comma separated items and that's wrong. I tested it. Returned should be items separated by comma. Until now it was wrong - separated by comma :) So it's a fix :)